charts/argo-cd/templates/argocd-configs/argocd-ssh-known-hosts-cm.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.server.enabled }}
```

Comment:
There is something extra or missing here with the `and`. Anyway I think we should remove the `server.enabled` configuration from all the places.

Reply:
.Values.server.enabled removed completely per above comment.